entrypoint.sh

```diff
@@ -12,7 +12,7 @@ cd $(dirname $0)
 
 if [[ ! -z $1 ]]; then
     if [[ "$1" == "listener" ]]; then
-        exec python3 -m listener.upload_listener
+        exec python3 -m listener.listener
```

Comment:
Shouldn't this still run the old listener?

Reply:
yep, my old debug change :D